config/7.1.0/securebanking/ig/scripts/groovy/ValidateDebtorAccountInRS.groovy

```diff
@@ -0,0 +1,53 @@
+def fapiInteractionId = request.getHeaders().getFirst("x-fapi-interaction-id");
+if(fapiInteractionId == null) fapiInteractionId = "No x-fapi-interaction-id"
+SCRIPT_NAME = "[ValidateDebtorAccountInRS] (" + fapiInteractionId + ") - "
+
+logger.debug(SCRIPT_NAME + "Running...")
+def method = request.method
+
+
+switch(method.toUpperCase()) {
+
+    case "POST":
+        def debtorAccount = request.entity.getJson().Data.Initiation.DebtorAccount
+        if(debtorAccount !=null){
+            def requestURI = routeArgRsBaseURI + "/backoffice/accounts/search/findByAccountIdentifiers" +
+                    "?name=" + debtorAccount.Name + "&identification=" + debtorAccount.Identification +
+                    "&schemeName=" + debtorAccount.SchemeName
+
+            logger.debug(SCRIPT_NAME + " findByAccountIdentifiers raw request uri: " + requestURI)
+            Request RSRequest = new Request()
+            RSRequest.setUri(requestURI)
+            RSRequest.setMethod('GET')
+            if (request.headers.get("x-fapi-financial-id") != null) {
+                RSRequest.putHeaders(request.headers.get("x-fapi-financial-id"))
+            }
```

Comment:
this should be able to be removed, you just need to configure the ScriptableFilter to use the `ForgeRockClientHandler`

Reply:
Ah, I thought this was the `x-fapi-interaction-id`

Why do we want to send the `x-fapi-financial-id` this has been removed from the OB spec

Reply:
It will be better create an issue to update the config to configure the `ScriptableFilter` to use the `ForgerRockClientHandler` and make a clean up through all existing scripts.
What is your thoughts?

Reply:
May as well fix this new filter that your adding then tackle any other calls to the backoffice API later if you think those are broken, as it is a quick fix.